### DISCUSSION TITLE
html > docx: unordered list style ( ul/@style= list-style-type / ul/@type is not preserved

### DISCUSSION THREAD

**Discussion by zsofiamacho** [jgm/pandoc#9621] [Q&A]
Hi,

I can't get the styling for **unordered** lists work in case of a html > docx conversion. **Ordered** lists work just fine. I tried both the "old" and the "new" unordered lists styling as per w3 ( https://www.w3docs.com/learn-html/html-lists.html ) Could you please help me with what I'm doing wrong?

`````
<html>
      <body>
               <p>works fine</p>
		<ol type="I">
			<li>roman first</li>
			<li>roman second</li>
			<li>roman third</li>
		</ol>
		<p>doesn't work (old html markup)</p>
		<ul type="square">
			<li>expecting square</li>
			<li>expecting square</li>
			<li>expecting square</li>
		</ul>
		<p>doesn't work (new html markup)</p>
		<ul style="list-style-type: square;">
			<li>expecting square</li>
			<li>expecting square</li>
			<li>expecting square</li>
		</ul>
	</body>
</html>
`````
My output is:
- roman numerals (as expected)
- disc (expected: square)
- disc (expected: square)
[output.docx](https://github.com/jgm/pandoc/files/14836179/output.docx)

**Comment by jgm** [comment]
We don't have any way to represent different unordered list markers in the AST, so this will just be lost in translation.  In general, pandoc is not designed to translate these stylistic details.

**Reply by zsofiamacho** [reply]
Okay, thanks, good to know!
